book/src/chap20.asciidoc

```diff
@@ -260,10 +259,10 @@ julia> typeof(z)
 Array{Float64,2}
 ----
 
-The type of this matrix is an array holding floating points and having 2 dimensions.
-(((dimension)))
+O tipo desta matriz é um array que guarda pontos flutuantes com duas dimensões.
```

Comment:
Acho que, talvez seja melhor preservar _lista_ já que em nenhum momento o livro diferenciou uma lista de um array ~talvez por causa da tradução pt-br~. Se não for o caso, desconsidere.
```suggestion
O tipo desta matriz é uma lista que guarda pontos flutuantes com duas dimensões.
```